checks/contributors.go

```diff
@@ -34,17 +35,23 @@ func init() {
 
 // Contributors run Contributors check.
 func Contributors(c *checker.CheckRequest) checker.CheckResult {
-	rawData, err := raw.Contributors(c.RepoClient)
+	rawData, err := raw.Contributors(c)
 	if err != nil {
 		e := sce.WithMessage(sce.ErrScorecardInternal, err.Error())
 		return checker.CreateRuntimeErrorResult(CheckContributors, e)
 	}
 
-	// Return raw results.
-	if c.RawResults != nil {
-		c.RawResults.ContributorsResults = rawData
+	// Set the raw results.
+	pRawResults := getRawResults(c)
+	pRawResults.ContributorsResults = rawData
+
+	// Evaluate the probes.
+	findings, err := evaluateProbes(c, pRawResults, probes.Contributors)
```

Comment:
note: keep an eye on https://github.com/ossf/scorecard/pull/3409 which will remove this function